app/src/main/java/com/example/parkingsystemjava/database/ReservationDataBase.java

```diff
@@ -0,0 +1,42 @@
+package com.example.parkingsystemjava.database;
+
+import com.example.parkingsystemjava.ReservationObject.Reservation;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ReservationDataBase {
+    private static ReservationDataBase reservationDataBase;
+    private final HashMap<String, ArrayList<Reservation>> reservationHashMap;
```

Comment:
please use interfaces not implementations, instead of HasMap use Map instead of ArrayList use List

Reply:
Done